.code-samples.meilisearch.yaml

```diff
@@ -853,60 +901,57 @@ settings_guide_typo_tolerance_1: |-
         "minWordSizeForTypos": {
           "twoTypos": 12
         },
-        "disableOnAttributes": [
-          "title"
-        ]
+        "disableOnAttributes": ["title"]
       }'
 updating_guide_check_version: |-
   curl \
-    -X GET 'http://127.0.0.1:7700/version' \
+    -X GET 'http://localhost:7700/version' \
     -H 'Authorization: Bearer apiKey'
 updating_guide_get_displayed_attributes_new: |-
   # whenever you see {index_uid}, replace it with your index's unique id
   curl \
-    -X GET 'http://127.0.0.1:7700/indexes/{index_uid}/settings/displayed-attributes' \
+    -X GET 'http://localhost:7700/indexes/{index_uid}/settings/displayed-attributes' \
     -H 'Authorization: Bearer apiKey'
 updating_guide_get_displayed_attributes_old: |-
   # whenever you see {index_uid}, replace it with your index's unique id
   curl \
-    -X GET 'http://127.0.0.1:7700/indexes/{index_uid}/settings/displayed-attributes' \
-    -H 'X-Meili-API-Key: apiKey'
+    -X GET 'http://localhost:7700/indexes/{index_uid}/settings/displayed-attributes' \
+    -H 'X-Meili-API-Key: apiKey'    
```

Comment:
Are the spaces after `apiKey'` required?

Reply:
Nope